test/sequential/test-child-process-pass-fd.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
+
+const N = 80;
```

Comment:
I suppose you could skip if `'67'.includes(process.config.variables.arm_version)`, that would filter out the ARMv6 and ARMv7 buildbots (i.e., the rpis.)